Missing `tableauhyperapi.h` during pip install

When performing a clean install of pantab from pip (pip install tableauhyperapi), the extension module breaks.

Full traceback below:

```
$ pip install pantab
Collecting pantab
  Using cached https://files.pythonhosted.org/packages/e3/82/e3844ce038e52f9927f43f5ab94984f9bbfb6f6dee25ea1fbd6e4b40eeae/pantab-0.1.0.tar.gz
Requirement already satisfied: pandas in /home/akos/akos_venv/lib/python3.6/site-packages (from pantab) (0.25.3)
Requirement already satisfied: numpy>=1.13.3 in /home/akos/akos_venv/lib/python3.6/site-packages (from pandas->pantab) (1.17.4)
Requirement already satisfied: python-dateutil>=2.6.1 in /home/akos/akos_venv/lib/python3.6/site-packages (from pandas->pantab) (2.8.1)
Requirement already satisfied: pytz>=2017.2 in /home/akos/akos_venv/lib/python3.6/site-packages (from pandas->pantab) (2019.3)
Requirement already satisfied: six>=1.5 in /home/akos/akos_venv/lib/python3.6/site-packages (from python-dateutil>=2.6.1->pandas->pantab) (1.13.0)
Building wheels for collected packages: pantab
  Building wheel for pantab (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/akos/akos_venv/bin/python3.6 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-p1jbd85j/pantab/setup.py'"'"'; __file__='"'"'/tmp/pip-install-p1jbd85j/pantab/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-o3f3iwl1 --python-tag cp36
       cwd: /tmp/pip-install-p1jbd85j/pantab/
  Complete output (28 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.6
  creating build/lib.linux-x86_64-3.6/benchmarks
  copying benchmarks/__init__.py -> build/lib.linux-x86_64-3.6/benchmarks
  copying benchmarks/custom.py -> build/lib.linux-x86_64-3.6/benchmarks
  copying benchmarks/benchmarks.py -> build/lib.linux-x86_64-3.6/benchmarks
  creating build/lib.linux-x86_64-3.6/pantab
  copying pantab/__init__.py -> build/lib.linux-x86_64-3.6/pantab
  copying pantab/_writer.py -> build/lib.linux-x86_64-3.6/pantab
  copying pantab/_reader.py -> build/lib.linux-x86_64-3.6/pantab
  copying pantab/_types.py -> build/lib.linux-x86_64-3.6/pantab
  creating build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/__init__.py -> build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/test_writer.py -> build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/conftest.py -> build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/test_roundtrip.py -> build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/test_reader.py -> build/lib.linux-x86_64-3.6/pantab/tests
  running build_ext
  building 'libwriter' extension
  creating build/temp.linux-x86_64-3.6
  creating build/temp.linux-x86_64-3.6/pantab
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/home/akos/akos_venv/include -I/usr/local/include/python3.6m -c pantab/_writermodule.c -o build/temp.linux-x86_64-3.6/pantab/_writermodule.o
  pantab/_writermodule.c:6:29: fatal error: tableauhyperapi.h: No such file or directory
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for pantab```

Did some investigation of this issue and found "tableauhyperapi.h" wasn't being copied when sdist was run, and therefore pip install pantab was throwing a missing file error: "pantab/_writermodule.c:6:29: fatal error: tableauhyperapi.h: No such file or directory". 

Came across this comment: https://github.com/pypa/packaging-problems/issues/84#issuecomment-384520493 which suggested updating MANIFEST.in to include "recursive-include pantab *.h". After adding this I can see sdist now picks up the missing header:

```
...
copying pantab/_types.py -> pantab-0.1.0/pantab                                                                                         
copying pantab/_writer.py -> pantab-0.1.0/pantab                                                                                          
copying pantab/_writermodule.c -> pantab-0.1.0/pantab                                                                              
copying pantab/tableauhyperapi.h -> pantab-0.1.0/pantab           
...
```

Install runs correctly after rebuild: 

```
...
Successfully built pantab
Installing collected packages: six, python-dateutil, pytz, numpy, pandas, pantab
Successfully installed numpy-1.17.4 pandas-0.25.3 pantab-0.1.0 python-dateutil-2.8.1 pytz-2019.3 six-1.13.0
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,2 +1,3 @@
 include README.md
 include LICENSE.txt
+recursive-include pantab *.h

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,3 +1,2 @@
 include README.md
 include LICENSE.txt
-recursive-include pantab *.h

--- a/pantab/newsfragments/40.bug
+++ b/pantab/newsfragments/40.bug
@@ -1,1 +1,1 @@
-Added "package_data={"": ["*.h"]}" to main setup call so that header files are correctly copied during build.
+Fixed issue where source installations would error with `fatal error: tableauhyperapi.h: No such file or directory`

--- a/pantab/newsfragments/40.bug
+++ b/pantab/newsfragments/40.bug
@@ -1,0 +1,1 @@
+Added "package_data={"": ["*.h"]}" to main setup call so that header files are correctly copied during build.

--- a/setup.py
+++ b/setup.py
@@ -50,6 +50,7 @@ setup(
     ],
     keywords="tableau visualization pandas dataframe",
     packages=find_packages(exclude=["samples", "tests"]),
+    package_data={"": ["*.h"]},
     python_requires=">=3.6",
     install_requires=["pandas"],
     extras_require={"dev": ["pytest"]},

